packages/events/README.md

```diff
@@ -0,0 +1,33 @@
+#  @pixi/events
+
+This packages implements the Federated Events API, the plumbing behind the propagation of UI events into the PixiJS
+scene graph.
+
+## Installation
+
+```bash
+npm install @pixi/events
+```
+
+## Usage
+
+The `pixi.js` and `pixi.js-legacy` bundles add the `EventSystem` to renderers by default.
+
+```ts
+import { EventSystem } from '@pixi/events';
+
+renderer.addSystem(EventSystem, 'events');
+renderer.render(stage);// Render stage so that it becomes the root target for UI events
```

Comment:
Is it required to render once first before adding listeners?

Also maybe use Application (or new Renderer) here for a full working example.

Reply:
Not for registering listeners, but for actually having the events fire - yes (the renderer doesn't know about the stage)